hosts/Pliant.Benchmarks/Program.cs

```diff
@@ -8,7 +8,10 @@ class Program
     {
         static void Main(string[] args)
         {
-            var summary = BenchmarkRunner.Run<ParsingBenchmarks>();
```

Comment:
Need to figure out how to run multiple benchmarks in one go so we don't have a lot of thrashing on this file